### ISSUE TITLE
manual import of stack.py interfers with unit testing

### ISSUE THREAD

**Issue by jan--f** [SUSE/DeepSea#168]
Is there a way of rewriting this line
https://github.com/SUSE/DeepSea/blob/master/srv/modules/runners/push.py#L59

I was starting a unit test suite for push.py but in a unit test, this import is not present.

**Comment by jan--f** [comment]
@swiftgist how about this:
```
cur_file_path = os.path.dirname(os.path.realpath(__file__))
stack_path = os.path.abspath('{}/../pillar/stack.py'.format(cur_file_path))
stack = imp.load_source('pillar.stack', stack_path)
```
Not pretty but not much worse and it works.

**Comment by jan--f** [comment]
For a proposed "fix" see #169 
This makes it at least a bit more flexible.

**Comment by jan--f** [comment]
clsoing. Fixed by #169
